src/timesteppers.jl

```diff
@@ -277,6 +279,89 @@ function stepforward!(sol, clock, ts::FilteredRK4TimeStepper, equation, vars, pa
   return nothing
 end
 
+# ------
+# LSRK(5)4
+# ------
+"""
+    struct LSRK54TimeStepper{T} <: AbstractTimeStepper{T}
+
+A 4th-order 5-stages 2-storage Runge-Kutta timestepper for time-stepping
+`∂u/∂t = RHS(u, t)` via:
+```
+S² = 0
+for i = 1:5
+  S² = Aᵢ * S² + dt * RHS(uⁿ, t₀ + Cᵢ * dt)
+  uⁿ = uⁿ + Bᵢ * S²
+end
+uⁿ⁺¹ = uⁿ
+```
+
+where `Aᵢ`, `Bᵢ`, and `Cᵢ` are the A, B, C coefficients from LSRK tableau
+table at the ``i``-th stage. For details, please refer to
+
+> Carpenter, M. H. and Kennedy, C. A. (1994). Fourth-order 2N-storage Runge–Kutta schemes,
+  Technical Report NASA TM-109112, NASA Langley Research Center, VA.
+"""
+struct LSRK54TimeStepper{T,V} <: AbstractTimeStepper{T}
+   S² :: T
+  RHS :: T
+    A :: V
+    B :: V
+    C :: V
+end
+
+"""
+    LSRK54TimeStepper(equation::Equation, dev::Device=CPU())
+
+Construct a 4th-order 5-stages low storage Runge-Kutta timestepper for `equation` on device `dev`.
+"""
+function LSRK54TimeStepper(equation::Equation, dev::Device=CPU())
+  @devzeros typeof(dev) equation.T equation.dims S² RHS
+  
+  T = equation.T
+  A = T[0.0, 
+        -567301805773/1357537059087, 
+        -2404267990393/2016746695238, 
+        -3550918686646/2091501179385, 
+        -1275806237668/842570457699]
+  B = T[1432997174477/9575080441755, 
+        5161836677717/13612068292357,
+        1720146321549/2090206949498,
+        3134564353537/4481467310338,
+        2277821191437/14882151754819]
+  C = T[0.0, 
+        1432997174477/9575080441755,
+        2526269341429/6820363962896,
+        2006345519317/3224310063776,
+        2802321613138/2924317926251]
```

Comment:
```suggestion
  A = T[0, 
        -567301805773//1357537059087, 
        -2404267990393//2016746695238, 
        -3550918686646//2091501179385, 
        -1275806237668//842570457699]

  B = T[1432997174477//9575080441755, 
        5161836677717//13612068292357,
        1720146321549//2090206949498,
        3134564353537//4481467310338,
        2277821191437//14882151754819]
  C = T[0, 
        1432997174477//9575080441755,
        2526269341429//6820363962896,
        2006345519317//3224310063776,
        2802321613138//2924317926251]
```

since

```Julia
julia> typeof(2//3)
Rational{Int64}

julia> typeof(2/3)
Float64
```